lib/utils.js

```diff
@@ -2,7 +2,9 @@
 
 // helper for convenient testing
 exports.require = (name) => {
-    const importedModule = require(require.resolve(name, {paths: [process.cwd()]})); // Require module using the absolute path
+    const defaultPathsToResolve = require.resolve.paths(name);
```

Comment:
It just returns default path the plugin can be resolved to?

Reply:
Yes, and I can actually pass just an empty string there. But I decided to pass the package name just in case if something changed in next nodejs versions.